plugins/inputs/dirmon/README.md

```diff
@@ -0,0 +1,51 @@
+# tail Input Plugin
```

Comment:
Looks like the README wasn't really edited. I gave feedback to @reimda that we may want a clearer name for this plugin besides dirmon. I suggested simply `inputs.directory`.

Reply:
We had a scenario where backlog files are processed when dirmon is restarted after  a network outage or similar incident that crashes the application. Upon successfully processing the backlog, dirmon stops scanning the directory for real time events. Yesterday, I resolved this issue. I am adding the code in dirmon.go.  You can find it at 
func (ddo DirDefObject) FileProcessor(id int).

The plugin is named "dirmon" because it monitors the directory where we expect files to be available for processing. These files are further processed to extract the metrics etc.